understanding-ibc-denoms/index.md

```diff
@@ -0,0 +1,400 @@
+---
+parent:
+  title: Understanding IBC Denoms with Gaia
+order: 0
+description: Send tokens with IBC and trace your Denom, understand how Denoms work.
+---
+
+# Understanding IBC Denoms with Gaia
+
+In this tutorial, you will learn how to use IBC denoms with Gaia, the Cosmos Hub software.
+
+Explore the low-level details of what it means to identify a token received using IBC to your account, find out which chain the token came from, contact that chain, and verify that it exists and is still running.
+
+Imagine that you've received a new `ibc/` token on a blockchain:
+
+`1000000ibc/27A6394C3F9FF9C9DCF5DFFADF9BB5FE9A37C7E92B006199894CF1824DF9AC7C,100000000000samoleans,99999977256stake`
+
+Just like `samoleans` or `stake`, `ibc/27A639...` is the denomination (denom) of the token received from IBC, but `27A639...` is a hash of the denom, IBC port, and channel.
+
+Why is `27A639...` a hash? Paths that track the token on multiple hops from other blockchains to your account would be unbearably long. The Cosmos SDK has a 64-character limit on the denomination of the token.
+
+The tradeoff is that you must query the node to find out what the actual path and denomination is. This node query is called the `denomtrace`. 
+
+In Cosmos SDK, the Gaia daemon `gaiad` runs a full node of the gaia application and provides a gRPC interface. The default port is  9090. In this tutorial, you will learn to query this interface directly. 
+
+For now, follow along with the `gaiad` subcommands.
+
+```bash
+gaiad q ibc-transfer denom-trace 27A6394C3F9FF9C9DCF5DFFADF9BB5FE9A37C7E92B006199894CF1824DF9AC7C --node tcp://localhost:26557
+denom_trace:
+  base_denom: samoleans
+  path: transfer/channel-0
+```
+
+From this command, you now know that there is an IBC port `transfer` and channel `channel-0`. But to know the IBC light client behind the port and channel, you need to perform another query. 
+
+Why is it called a light client? Because it is a light client of the _other_ chain, keeping track of its blockhashes. The `ibc channel client-state transfer` command  explains the details of the denom path. 
```

Comment:
```suggestion
Why is the path in IBC called a light client? Because it is a light client of the _other_ chain, keeping track of its blockhashes. The `ibc channel client-state transfer` command  explains the details of the denom path. 
```

Reply:
I wasn't sure what "it" was in the sentence, I think we are describing the path in IBC (guessing here!)